internal/workflow_testsuite.go

```diff
@@ -539,6 +539,19 @@ func (e *TestWorkflowEnvironment) SetIdentity(identity string) *TestWorkflowEnvi
 	return e
 }
 
+// SetDetachedChildWait, if true, will make ExecuteWorkflow wait on all child
+// workflows to complete even if their close policy is set to abandon or request
+// cancel. If false, ExecuteWorkflow will wait on all children to complete
+// regardless of detached child. In situations with interminable cron-based
```

Comment:
This sentence is unclear to me:
> regardless of detached child
Can you please clarify?

Reply:
It means, regardless of whether the child is detached or not, `ExecuteWorkflow` will wait for all to complete. This is the default behavior today and the problematic one.

Reply:
It is a tricky thing to describe. Perhaps something like this might be clearer?
> If false, ExecuteWorkflow will block until all attached child workflows have completed and all detached child workflows have ran only once. This is useful when testing endless cron-based child workflows, as without it a detached child workflow would run forever.

Reply:
Clarified